code/modules/goals/goal_department.dm

```diff
@@ -1,12 +1,12 @@
 /datum/department
-	var/name
-	var/flag
-	var/list/goals
+	//var/name
+	//var/flag
+	//var/list/goals
 	var/min_goals = 1
 	var/max_goals = 2
 
 /datum/department/proc/Initialize()
```

Comment:
You should really centralize these definitions with your new ones so they're easier to find.